mona_sdk/client_util.py

```diff
@@ -75,3 +75,12 @@ def keep_message_after_sampling(context_id, sampling_rate):
     else:
         # TODO(Nemo): Allow getting seed from the user for random.random().
         return random.random() <= sampling_rate
+def get_dict_result(success, data, error_message):
+    if isinstance(error_message, str):
+        error_message = error_message.replace("\"", "")
```

Comment:
please add comment why this removal of double quotes is needed - I'd expect us to just keep the original string, is this just to make things prettier? I'm afraid of losing important information in strings that were dumped from json, for example...

Reply:
I'd prefer to not do this replacement if there is no real reason actually... It might lose important information